gnosis/gnosis-spec.ini

```diff
@@ -277,7 +277,10 @@ proxy_storage: _
     andBool G >=Int 10000 //
 
 [encodeTransactionData]
-output: _ => _
+output: _ => #encodeArgs( #bytes(
+                #parseHexWord("0x19") : #parseHexWord("0x1")
+                : #encodeArgs(#bytes32(DOMAIN_SEPARATOR), #bytes32(SAFE_TX_HASH))
+             ))
```

Comment:
So, the output will be the concatenation of the following bytes?
`bytes32(32)  bytes32(66)  bytes1(0x19)  bytes1(0x1)  bytes32(DOMAIN_SEPARATOR)  bytes32(SAFE_TX_HASH)  bytes30(0)`
and the total size is  160 = 32 + 32 + 1 + 1 + 32 + 32 + 30 ?

In any case, could you put this information as a comment?

Reply:
done